stl/inc/xcharconv_tables.h

```diff
@@ -0,0 +1,114 @@
+// xcharconv_tables.h internal header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _XCHARCONV_TABLES_H
+#define _XCHARCONV_TABLES_H
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#if !_HAS_CXX17
+#error The contents of <charconv> are only available with C++17. (Also, you should not include this internal header.)
+#endif // !_HAS_CXX17
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+#include <cstdint>
+
+// For general precision, we can use lookup tables to avoid performing trial formatting.
+
+// For a simple example, imagine counting the number of digits D in an integer, and needing to know
+// whether D is less than 3, equal to 3/4/5/6, or greater than 6. We could use a lookup table:

```

Comment:
I think we are talking about digits here
```suggestion
// whether D has less than 3, equal to 3/4/5/6, or greater than 6 digits. We could use a lookup table:
```

Reply:
No, I don't think so, D is the result operand of counting digits, not the source.

Reply:
@AlexGuteniev is correct, the comment is correct as written.